tests/crypto/rand32/testcase.yaml

```diff
@@ -16,3 +16,6 @@ tests:
     filter: CONFIG_ENTROPY_HAS_DRIVER
     tags: crypto entropy random security
     min_ram: 16
+  crypto.rand32.user:
+    tags: crypto random security userspace
+    min_ram: 16
```

Comment:
maybe you should put 'CONFIG_TEST_USERSPACE=y' at here using extra_configs options, if you added it in prj.conf, it suits for all testcases in this test.

Reply:
The concern from Peng is right. And I check the coverage, it seems like by your test case doesn't cover rand32_handlers.c, could you please refine it again? Thanks!

Reply:
Thank you for the suggestion. Yes, it is inappropriate to run the same case twice in kernel and userspace, so I split the case to two to run the new one in userspace. Is this appropriate?